src/Illuminate/Database/Query/Grammars/Grammar.php

```diff
@@ -629,6 +629,18 @@ protected function compileJsonLength($column, $operator, $value)
         throw new RuntimeException('This database engine does not support JSON length operations.');
     }
 
+    /**
+     * Compile a "where fulltext" clause.
+     *
+     * @param  \Illuminate\Database\Query\Builder  $query
+     * @param  array  $where
+     * @return string
+     */
+    public function whereFulltext(Builder $query, $where)
+    {
+        throw new RuntimeException('This database engine does not support fulltext search operations.');
```

Comment:
This means for other engines where fulltext search operations have not been implemented we still get this exception🤨.

Reply:
You just don't use this method on other engines. It doesn't makes sense to use a method that's not supported.

Reply:
Yeah. But since not all of them are covered now. For instance, SQL Server supports fulltext search but it's not been added. I think, we could modify the exception message a little. Something in the lines of ```This database engine does not support fulltext search operations or there is no current implementation of it yet. Kindly revert to external packages if that's the case```. Too long, tho'

Reply:
You can send in a PR if you want mssql support 👍